docs/api.rst

```diff
@@ -5298,6 +5298,7 @@ Exporting History Events
    :statuscode 400: Provided JSON is in some way malformed
    :statuscode 409: No user is logged in or failure at event export.
    :statuscode 500: Internal rotki error
+   :statuscode 502: Couldn't fetch prices for all the events due to being rate limited.
```

Comment:
Shouldnt't the async change also be mentioned in the docs?

Reply:
added